### PR TITLE
Pm 2021 09 add category to list

### PR DESCRIPTION
**Pull request by phillimorland**

### CODE REVIEW

File: theme/spacings.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ export const BORDERWIDTH = {
 
 export const BORDERRADIUS = {
   none: 0,
-  base: 15,
+  base: 6,
+  md: 12,
   container: 24
```
Comment (phillimorland) [inline theme/spacings.js:16]: adjust to be inline with zeplin

**Comment by phillimorland** [comment]
@goapunk @fuzzylogic2000 should we have start adding tests everytime we work on a view do you think?

**Review by fuzzylogic2000 — APPROVED** [review]
Looks good!

**Comment by fuzzylogic2000** [comment]
> @goapunk @fuzzylogic2000 should we have start adding tests everytime we work on a view do you think?

I would say yes! <3 tests! But let's do that starting from the next PR. We need to merge @goapunk s PR first anyway. :)

**Pull request merged**